cunumeric/runtime.py

```diff
@@ -118,12 +131,12 @@
 
 
 class Runtime(object):
-    def __init__(self, legate_context):
+    def __init__(self, legate_context: LegateContext):
```

Comment:
don't you need a return type?

Reply:
https://github.com/nv-legate/cunumeric/pull/428/commits/13fb8be4bdfcbe2af4edd7a469e216dc6805a2ca (not sure why mypy was not complaining, though)